plugins/inputs/gnmi/handler.go

```diff
@@ -164,7 +164,11 @@ func (h *handler) handleSubscribeResponseUpdate(acc telegraf.Accumulator, respon
 	prefix := newInfoFromPath(response.Update.Prefix)
 
 	// Add info to the tags
-	headerTags["source"], _, _ = net.SplitHostPort(h.address)
+	var err error
+	headerTags["source"], _, err = net.SplitHostPort(h.address)
+	if err != nil {
+		h.log.Debugf("unable to parse address %s: %v", h.address, err)
```

Comment:
Same question here, error? warning? or print once?

Reply:
I decided to log this at `Error` level which matches other logging calls in this function